package.json

```diff
@@ -22,6 +22,7 @@
     "build-plugins": "./bin/plugin/cli.js build-plugins",
     "test-plugins": "./bin/plugin/cli.js test-plugins",
     "test-plugins-multisite": "./bin/plugin/cli.js test-plugins --sitetype=multi",
+    "get-plugin-version": "./bin/plugin/cli.js get-plugin-version",
```

Comment:
This script is fine to add for testing purposes, but is not necessary since you're not actually using it during the workflow and are calling the CLI script directly. Note that if you wanted to test this output, you need to add an extra `--` before the options that you want to pass to the CLI, like:

```
npm run get-plugin-version -- --slug="webp-uploads"
```

Reply:
Thanks @joemcgill for the feedback. If i use above mention command in workflow then it also return other details which we don't needed. 

```
> performance@ get-plugin-version /Users/abc/performance
> ./bin/plugin/cli.js get-plugin-version "--slug=webp-uploads"

1.0.0
```

Reply:
Yeah, the way you're doing it now is fine. I just noticed that the shortcut to the script in the `package.json` file here isn't being used directly.

Reply:
Thanks @joemcgill. In https://github.com/WordPress/performance/pull/710/commits/092a37fabade1740c135ee8c8902fdc81fc82da2 remove command from `package.json`